website/static/css/style.css

```diff
@@ -437,6 +437,10 @@ select {
     position: relative;
 }
 
+span.tag.tag-big.tag-container {
```

Comment:
After discussing with Caner, all the word break css will be put in base.css from style guide. So I delete this! Never mind!